systemtest/benchtest/expvar.go

```diff
@@ -74,3 +79,32 @@ func queryExpvar(out *expvar) error {
 	defer resp.Body.Close()
 	return json.NewDecoder(resp.Body).Decode(out)
 }
+
+// WaitUntilServerInactive blocks until one of the conditions occurs:
+// * APM Server output active events are lower than `warmupInactive`.
+// * HTTP call returns with an error
+// * Context is done.
+func WaitUntilServerInactive(ctx context.Context) error {
+	errChan := make(chan error)
+	go func() {
+		var result expvar
+		defer close(errChan)
+		for {
+			select {
+			case <-ctx.Done():
+				errChan <- ctx.Err()
+				return
+			default:
+			}
+			if err := queryExpvar(&result); err != nil {
+				errChan <- err
+				return
+			}
+			if result.ActiveEvents < int64(*inactiveThreshold) {
```

Comment:
What's the reason for making this configurable, and > 0? Can we just wait for the server to be completely inactive?

Reply:
I think I may have overcomplicated this more than necessary. It should be possible to just wait until 0 rather than the threshold.